pyaedt/icepak.py

```diff
@@ -4277,8 +4277,7 @@ def assign_free_opening(
             Default is ``"AmbientRadTemp"``.
         flow_type : int or str, optional
             Prescribed radiation flow type at the boundary. Available options are ``"Pressure"``,
-            ``"Velocity"`` and ``"Mass Flow"``.  Also, integers can be used, respectively 0,1 and 2.
-            Default is ``0``.
+            ``"Velocity"`` and ``"Mass Flow"``.
```

Comment:
```suggestion
            ``"Velocity"``, and ``"Mass Flow"``.
```
We use the Oxford comma, which says to put a comma before the comma and the "and" that precedes the last list item.